axon-framework/events/event-bus-and-event-store.md

```diff
@@ -67,7 +67,7 @@ There are multiple `EventStorageEngine` implementations available:
 
 #### `JpaEventStorageEngine`
 
-The `JpaEventStorageEngine` stores events in a JPA-compatible data source. The JPA event store stores events in entries. These entries contain the serialized form of an event, as well as some fields where metadata is stored for fast lookup of these entries. To use the `JpaEventStorageEngine`, you must have the JPA \(`javax.persistence`\) annotations on your classpath.
+The `JpaEventStorageEngine` stores events in a JPA-compatible data source. The JPA event store stores events in entries. These entries contain the serialized form of an event, as well as some fields where metadata is stored for fast lookup of these entries. To use the `JpaEventStorageEngine`, you must have the JPA \(`jakarta.persistence`\) annotations on your classpath. The old, javax JPA \(`javax.persistence`\) annotations are also still supported. This will use an instance of the `JpaEventStorageEngine` using the `javax` namespace.
```

Comment:
I may be wrong, but I think it should be this, right?

```suggestion
The `JpaEventStorageEngine` stores events in a JPA-compatible data source. The JPA event store stores events in entries. These entries contain the serialized form of an event, as well as some fields where metadata is stored for fast lookup of these entries. To use the `JpaEventStorageEngine`, you must have the JPA \(`jakarta.persistence`\) annotations on your classpath. The old, javax JPA \(`javax.persistence`\) annotations are also still supported. This will use an instance of the `JpaEventStorageEngine` using the `legacyjpa` namespace.
```